Dockerfile

```diff
@@ -80,7 +80,9 @@ RUN /bin/bash -c "mkdir -p /opt/odoo/var/{run,log,egg-cache}"
 # SM: Add Odoo sources (remove .git folder to reduce image size)
 WORKDIR /opt/odoo/sources
 RUN git clone https://github.com/odoo/odoo.git -b 9.0 odoo && \
-       rm -rf odoo/.git
+      cd odoo && \
+      git reset --hard 7592d85b1fe7e4e7cd4473d263d145380b8ed406 && \
```

Comment:
So actually, if you clone with "--depth 1" and then "git revert --hard <sha1>" for a commit which is not the most recent one, it fails with the following error message:
fatal: Could not parse object '<sha1>'
It means that if we update the Dockerfile either manually or automatically with the SHA1 of the most recent commit (which will trigger a build on Docker Hub), if a new commit is submitted before the image is built, the build will fail. We need to find a better solution.
For now, I think we can live with the full clone because the only difference is going to be the build time (image size is the same since we remove the .git folder anyway) and we don't really care about how long it takes to build the image... Agreed?